### PR TITLE
Payment API params renaiming

### PR DESCRIPTION
**Pull request by Artemkaaas**
Signed-off-by: artem.ivanov <artem.ivanov@dsr-company.com>

### CODE REVIEW

File: libindy/src/api/payments.rs [hunk 1]
```diff
@@ -169,29 +170,29 @@ pub type BuildPaymentReqCB = extern fn(command_handle: i32,
 ///   Note: this param will be used to determine payment_method
 ///
 /// #Returns
-/// utxo_json - parsed (payment method and node version agnostic) utxo info as json:
+/// receipts_json - parsed (payment method and node version agnostic) receipts info as json:
 ///   [{
-///      txo: <str>, // UTXO input
-///      paymentAddress: <str>, //payment address for this UTXO
-///      amount: <int>, // amount of tokens in this input
+///      receipt: <str>, // source input
```
Comment (jovfer) [inline libindy/src/api/payments.rs]: @Artemkaaas @vimmerru Is comment here correct?

File: libindy/src/api/payments.rs [hunk 2]
```diff
@@ -454,34 +455,31 @@ pub extern fn indy_list_payment_addresses(command_handle: i32,
...
 ///
 /// #Params
 /// wallet_handle: wallet handle
 /// submitter_did : DID of request sender
 /// req_json: initial transaction request as json
-/// inputs_json: The list of UTXO inputs as json array:
-///   ["input1", ...]
-///   Notes:
-///     - each input should reference paymentAddress
-///     - this param will be used to determine payment_method
```
Comment (jovfer) [inline libindy/src/api/payments.rs]: @Artemkaaas @vimmerru are we actually would like to remove the note about way to determine payment method?

File: libindy/src/api/payments.rs [hunk 3]
```diff
@@ -770,11 +771,11 @@ pub extern fn indy_build_payment_req(command_handle: i32,
 ///   Note: this param will be used to determine payment_method
 ///
 /// #Returns
-/// utxo_json - parsed (payment method and node version agnostic) utxo info as json:
+/// receipts_json - parsed (payment method and node version agnostic) receipts info as json:
 ///   [{
-///      txo: <str>, // UTXO input
-///      paymentAddress: <str>, //payment address for this UTXO
-///      amount: <int>, // amount of tokens in this input
+///      receipt: <str>, // source input
```
Comment (jovfer) [inline libindy/src/api/payments.rs]: @Artemkaaas @vimmerru Is comment here correct?

File: libindy/src/api/payments.rs (hunk 1 above)
Comment (vimmerru) [reply libindy/src/api/payments.rs]: Should be

```
receipt: <str>, // receipt that can be used for payment referencing and verification
```

File: libindy/src/api/payments.rs (hunk 2 above)
Comment (vimmerru) [reply libindy/src/api/payments.rs]: I suggest to keep it

File: libindy/src/api/payments.rs (hunk 3 above)
Comment (vimmerru) [reply libindy/src/api/payments.rs]: receipt: <str>, // receipt that can be used for payment referencing and verification

File: libindy/src/api/payments.rs [hunk 4]
```diff
@@ -49,13 +49,13 @@ pub type CreatePaymentAddressCB = extern fn(command_handle: i32,
...
-/// outputs_json: The list of UTXO outputs as json array:
+/// inputs_json: The list of payment sources as json array:
+///   ["source1", ...]
+///   Note that each source should reference payment address
+/// outputs_json: The list of outputs as json array:
 ///   [{
-///     paymentAddress: <str>, // payment address used as output
-///     amount: <int>, // amount of tokens to transfer to this payment address
+///     recipient: <str>, // payment address of recipient
+///     amount: <int>, // amount of tokens to transfer to this recipient
```
Comment (vimmerru) [inline libindy/src/api/payments.rs]: > amount of tokens

should be changed to just amount. Make sure that there is no "token" word also.

File: libindy/src/api/payments.rs [hunk 5]
```diff
@@ -544,12 +542,15 @@ pub extern fn indy_add_request_fees(command_handle: i32,
...
+/// in the future releases.
+///
 /// #Returns
-/// utxo_json - parsed (payment method and node version agnostic) utxo info as json:
+/// receipts_json - parsed (payment method and node version agnostic) receipts info as json:
 ///   [{
-///      txo: <str>, // UTXO input
-///      paymentAddress: <str>, //payment address for this UTXO
-///      amount: <int>, // amount of tokens in this input
+///      receipt: <str>, // receipt
```
Comment (vimmerru) [inline libindy/src/api/payments.rs]: receipt: <str>, // receipt that can be used for payment referencing and verification

File: cli/src/commands/common.rs [hunk 6]
```diff
@@ -94,7 +94,7 @@ pub mod load_plugin_command {
     command!(CommandMetadata::build("load-plugin", "Load plugin in Libindy")
                             .add_required_param("library", "Name of plugin (can be absolute or relative path)")
                             .add_required_param("initializer", "Name of plugin init function")
-                            .add_example("load-plugin library=libsovtoken initializer=sovtoken_init")
+                            .add_example("load-plugin library=libsovplugin initializer=sovplugin_init")
```
Comment (vimmerru) [inline cli/src/commands/common.rs]: We need to switch to libnullpay usage in samples instead of libsovplugin

**Pull request merged**